Misc/NEWS.d/next/Library/2019-07-21-20-59-31.bpo-37642.L61Bvy.rst

```diff
@@ -0,0 +1 @@
+Update :mod:`datetime.timezone` to get closer to 24:00
```

Comment:
I think it will be very unclear what this change is to someone reading it without having read the bug report. This specifically allows the *pure Python* implementation `datetime.timezone` to represent sub-minute offsets close to 24 hours.

I suggest wording like:

"Allowed the pure Python implementation of ``:class:`datetime.timezone` `` to represent sub-minute offsets close to minimum and maximum boundaries, specifically in the ranges (23:59, 24:00) and (-23:59, -24:00). Patch by Ngalim Siregar."

You may also want to add yourself to the [Misc/Acks](https://github.com/python/cpython/blob/master/Misc/ACKS) file as well.